src/components/AfterServerView.tsx

```
@@ -216,20 +217,13 @@ export default function AfterServerView({ server }: { server: string }) {
-                      sandbox="allow-popups allow-popups-to-escape-sandbox allow-same-origin allow-scripts"
-                    />
-                  </CardDescription>
-                </CardHeader>
-              </Card>
+              <iframe
+                src={`https://discord.com/widget?id=${discord}&theme=${resolvedTheme}`}
+                height="500"
+                allowTransparency={true}
+                className="rounded-lg max-md:w-full"
+                sandbox="allow-popups allow-popups-to-escape-sandbox allow-same-origin allow-scripts"
+              />
```

Biome (1.9.4)

[error] Provide a title attribute when using iframe elements.

Screen readers rely on the title set on an iframe to describe the content being displayed.

(lint/a11y/useIframeTitle)